libs/design-system/src/lib/Components/FAQ/FAQ.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+
+import { ReactComponent as ChevronRight } from '../../../../images/arrows/chevron-right.svg';
+import { ReactComponent as ChevronDown } from '../../../../images/arrows/chevron-down.svg';
+
+export interface IFAQ {
+  question: string;
+  answer: string;
+}
+
+export interface FAQProps {
+  faqList: IFAQ[];
+  variation: 'large' | 'small';
+}
+
+export function FAQ({ faqList, variation }: FAQProps) {
+  const [expanded, setExpanded] = useState(
+    new Array(faqList.length).fill(false)
+  );
+
+  const toggleExpansion = (index: number) => {
+    const newExpanded = [...expanded];
+    newExpanded[index] = !newExpanded[index];
+    setExpanded(newExpanded);
+  };
+
+  return (
+    <div className="bg-transparent">
+      {faqList.map((faq, i) => {
+        const itemExpanded = expanded[i];
+        const isAboveExpandedItem = expanded.at(i + 1);
+        const isLast = i === expanded.length - 1;
+
+        return (
+          <div
```

Comment:
We could use https://reach.tech/accordion for accessibility. A quick improvement would be making this div a button